src/main/java/edu/ucla/library/bucketeer/utils/CodeUtils.java

```diff
@@ -25,7 +29,7 @@ private CodeUtils() {
      * @return A message code in integer form
      * @throws IllegalArgumentException If the supplied string isn't a message code
      */
-    public static int getInt(final String aMessageCode) throws IllegalArgumentException {
+    public static int getInt(final String aMessageCode) {
```

Comment:
Just making sure this change was intentional -- Javadoc still has `@throws`.

Reply:
Yes, PMD's rules say runtime exceptions should not be in the method's signature, but can still be in the Javadocs (I think they're still useful for the developer to know even if they shouldn't be a part of the API). Cf. https://stackoverflow.com/questions/824217/should-methods-that-throw-runtimeexception-indicate-it-in-method-signature

Reply:
Ok, thanks.